### PR TITLE
Framework Restructure Changes

### PR DESCRIPTION
**Pull request by VishalHaibatpure**
## Description
</br>
 Brief description of the requested change.

## Checklist
- [ ] I have read the instructions mentioned in [Contribute to Code](/Contributing.md).
- [ ] I have read and understood the criteria described under [submitting changes](/Contributing.md#submitting-changes). 
- [ ] The title of the PR clearly describes the intent of the PR.
- [ ] This PR does not introduce any breaking changes to the code.

### CODE REVIEW

File: src/AzSK.Framework/Abstracts/AzCommandBase.ps1 [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+						{
+							throw [SuppressedException]::new("The current subscription has been configured with DevOps kit policy for the '$OrgName' Org, However the DevOps kit command is running with a different ('$($AzSKConfigData.PolicyOrgName)') Org policy. `nPlease review FAQ at: https://aka.ms/devopskit/orgpolicy/faq and correct this condition depending upon which context(manual,CICD,CA scan) you are seeing this error. If FAQ does not help to resolve the issue, please contact your Org policy Owner ($($SubOrgTag.Value)).",[SuppressedExceptionType]::Generic)
+							
+						}
+						else
+						{	
+							if(-not $Force)
+							{
+								$this.PublishCustomMessage("Warning: The current subscription has been configured with DevOps kit policy for the '$OrgName' Org, However the DevOps kit command is running with a different ('$($AzSKConfigData.PolicyOrgName)') Org policy. `nPlease review FAQ at: https://aka.ms/devopskit/orgpolicy/faq and correct this condition depending upon which context(manual,CICD,CA scan) you are seeing this error. If FAQ does not help to resolve the issue, please contact your Org policy Owner ($($SubOrgTag.Value)).",[MessageType]::Warning);
```
[the remark this answers is not in the document]
Comment (VishalHaibatpure) [reply src/AzSK.Framework/Abstracts/AzCommandBase.ps1]: All over the docs and code files we have DevOps Kit. with space. Will confirm with Manish if needed

File: src/AzSK.Framework/Helpers/ContextHelper.ps1 [hunk 2]
```diff
@@ -0,0 +1,175 @@
+class ContextHelper : EventBase
+{
+    static hidden [PSObject] $currentRMContext;
```
Comment (v-zhgup) [inline src/AzSK.Framework/Helpers/ContextHelper.ps1]: Should this be AzContextHelper?

Comment (VishalHaibatpure) [reply src/AzSK.Framework/Helpers/ContextHelper.ps1]: This class name is kept generic as it has generic methods which will be used by all framework classes.  We need to rename the properties inside this class to generic one like change CurrenRMContext --> CurrentContext etc. We will be doing it in next phase after merging this staging changes to develop

File: src/AzSK.AzureDevOps/Framework/Core/SVT/AzureDevOps/AzureDevOps.ServiceConnection.ps1 [hunk 3]
```diff
@@ -1,5 +1,5 @@
 Set-StrictMode -Version Latest 
-class ServiceConnection: SVTBase
+class ServiceConnection: AzSVTBase
```
[the remark this answers is not in the document]
Comment (VishalHaibatpure) [reply src/AzSK.AzureDevOps/Framework/Core/SVT/AzureDevOps/AzureDevOps.ServiceConnection.ps1]: Resolved. Changed ot SVTBase. JFI, We will be replacing/changing complete code of AzSK.AzureDevOps in next changes phase

File: src/AzSK.Framework/Abstracts/AzSVTCommandBase.ps1 [hunk 4]
```diff
@@ -0,0 +1,167 @@
...
+	{
+		#Disable AzureRM telemetry setting until scan is completed.
+		#This has been added to improve the performarnce of scan commands
+		#Telemetry will be re-enabled once scan is completed
+        Disable-AzDataCollection  | Out-Null
+
+    }
+    
+    hidden [void] CheckAndEnableAzureRMTelemetry()
+    {
```
[the remark this answers is not in the document]
Comment (VishalHaibatpure) [reply src/AzSK.Framework/Abstracts/AzSVTCommandBase.ps1]: Updated. There are many more RM variables and methods present in solution. We need to change it all once.

**Pull request merged**